tests/foreman/api/test_product.py

```diff
@@ -1,52 +0,0 @@
-# -*- encoding: utf-8 -*-
```

Comment:
By dropping this file we're basically dropping "all" API tests for Products in the z-stream branch, which makes me really sad. :/ How hard would it be to add some CRUD tests instead?